ingestion/src/levels.rs

```diff
@@ -75,16 +75,18 @@ pub enum Error {
 
 #[derive(Debug, Clone)]
 pub struct Level {
-    hlevel: i32,
+    hlevel: Option<i32>,
     hlevel_scale: Option<i32>,
+    hlevel_type: Option<String>,
```

Comment:
Why are these `Option`? And why is the new one `String`

Reply:
Yeah, `hlevel_scale` should not have been an Option in the first place. The new one is String because it's TEXT in stinfosys?

Reply:
I see your point, we can 'wash' the data from stinfosys better as we take it in instead of handling it in the second function. 
Level was also originally not a function because we checked for it not being null in the SQL... We worried this might mean we didn't get scales/unit for stuff where there is no default.

Reply:
> We worried this might mean we didn't get scales/unit for stuff where there is no default.

I think it's appropriate to replace `NULL` with `0`, right? It's consistent with the hlevel scheme's approach of `NULL` and `0` always being the same, and in this case I don't think it affects behaviour anyway

Reply:
resolved?